src/ambianic/webapp/flaskr.py

```diff
@@ -159,6 +161,99 @@ def get_status():
         resp = jsonify(response_object)
         return resp
 
+    # start the auth0 authentication process with creating the auth0 keys
+    @app.route('/api/auth/user-code', methods=['POST'])
```

Comment:
Too much code in the main flaskr file. Move auth management in its own python module with its own unit tests.

Reply:
As agreed, the premium auth management would be split into its own module in a follow-up PR. 

They are currently being covered with unit tests.

Reply:
Create an issue to make sure this discussion does not get lost.

Reply:
[This](https://github.com/ambianic/ambianic-edge/issues/352) issue has been created to handle this concern.